sensing/autoware_cuda_pointcloud_preprocessor/CMakeLists.txt

```diff
@@ -78,6 +80,38 @@ rclcpp_components_register_node(cuda_pointcloud_preprocessor
   EXECUTABLE cuda_pointcloud_preprocessor_node
 )
 
+################################################################################
+### cuda_downsample_filter ###
+cuda_add_library(cuda_voxel_grid_downsample_filter_lib SHARED
+  src/cuda_downsample_filter/cuda_voxel_grid_downsample_filter.cu
+)
+
+target_include_directories(cuda_voxel_grid_downsample_filter_lib SYSTEM PRIVATE
+  ${cuda_blackboard_INCLUDE_DIRS}
+  ${sensor_msgs_INCLUDE_DIRS}
+)
+
+# Targets
+ament_auto_add_library(cuda_voxel_grid_downsample_filter SHARED
+  src/cuda_downsample_filter/cuda_voxel_grid_downsample_filter_node.cpp
+)
+
+ament_target_dependencies (cuda_voxel_grid_downsample_filter # XXX: DEBUG
```

Comment:
What is the meaning of
`XXX: DEBUG`?

Reply:
Sorry, I left this code carelessly. I deleted here in the latest code.